.gitignore

```diff
@@ -1 +1,4 @@
+generated_python/
```

Comment:
`generated_python/` contains committed files. So, it should not be ignored.
I've committed this directory in order to avoid that people have to generate it themselves.

Reply:
Ah... I just noticed, that the folder generated_python contained changed files in `git status` so I added it to .gitignore. Turns out, the changed files are located in the __pycache__ subdirectory.
I updated the .gitignore file and pushed again!